insonmnia/miner/server.go

```diff
@@ -803,27 +811,36 @@ func (m *Miner) runGPUBenchGroup(benches []*pb.Benchmark) error {
 	for _, dev := range m.hardware.GPU {
 		for _, ben := range benches {
 			if ben.GetID() == bm.GPUCount {
-				dev.Benchmark[bm.GPUCount] = &pb.Benchmark{ID: ben.GetID(), Result: 1}
+				dev.Benchmarks[bm.GPUCount] = &pb.Benchmark{
+					ID:     ben.GetID(),
+					Code:   ben.GetCode(),
+					Result: 1,
```

Comment:
Stub?

Reply:
No, this benchmark measures an amount of GPUs for... hmm, one GPU unit.